var/spack/repos/builtin/packages/neurodamus-model/package.py

```diff
@@ -201,6 +211,22 @@ def setup_run_environment(self, env):
 fi
 
 # run with nrnivmodl in path
-set -x
-'{nrnivmodl}' -incflags '{incflags} '"$2" -loadflags '{loadflags} '"$3" "$1"
+set -xe
+
+
+if [ -n "{nrnivmodlcore_call}" ]; then
+    mkdir -p _core_mods
```

Comment:
_core_mods folder is intermediate and it should be removed at the end?

Also, if I pass non-existent dir then we get:

``
+ mkdir -p _core_mods
+ touch mod/neuron_only_mods.txt
touch: cannot touch ‘mod/neuron_only_mods.txt’: No such file or directory
```

I think this error message cam be bit more clear.

Reply:
`_core_mods` is temporary folder so I will create that under x86_64.

Purposefully I introduced an error in `mod/neuron_only_mods.txt`. I then revered this text file and removed x86_64 folder as we typically do. But I still was getting error and got confused. I then realised that there is `_core_mods/` at the top level and removing that dir I started in clean state. Hence I my suggestion to have _core_mods inside x86_64.